dirsrvtests/tests/suites/webui/__init__.py

```diff
@@ -0,0 +1,62 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+
+import pytest
+
+from lib389.utils import *
+from lib389.topologies import topology_st
+
+pytest.importorskip('playwright')
+
+def remove_instance_through_lib(topology):
+    log.info('Check and remove instance before starting tests')
+    if topology.standalone.exists():
+        topology.standalone.delete()
+
+
+def remove_instance_through_webui(topology, page):
+    frame = page.frame('cockpit1:localhost/389-console')
+
+    log.info('Check if instance exist')
+    if topology.standalone.exists():
+        log.info('Delete instance')
+        frame.wait_for_selector('#ds-action')
+        frame.click('#ds-action')
+        frame.click('#remove-ds')
+        frame.check('text=", I am sure." >> input[type="checkbox"]')
+        frame.click('//button[normalize-space(.)=\'Remove Instance\']')
+        frame.wait_for_selector("#no-inst-create-btn")
+        log.info('Instance deleted')
+    #frame.wait_for_selector("#no-inst-create-btn")
+
+    #assert frame.is_visible("#no-inst-create-btn")
+
+
+def setup_login(page):
+    page.set_viewport_size({"width": 1920, "height": 1080})
+    page.goto("http://localhost:9090/")
+
+    # We are at login page
+    page.fill('#login-user-input', 'root')
+    page.fill('#login-password-input', 'redhat')
```

Comment:
Found another issue with the CI container image. We don't have password set there. I don't like the idea of hardcoding the password, I'd rather generate it on runtime, change it before the start of the test and pass the value to the test, so it can use it. I'll send you the proposed changes.

Reply:
I also don't like the hardcoded value so I am glad there is another suggestion. Thanks, will check the changes.